containers/ecr-viewer/src/app/services/htmlTableService.ts

```diff
@@ -1,14 +1,15 @@
 import { RenderableNode, safeParse } from "@/app/utils/data-utils";
 import { parse, HTMLElement, Node, NodeType } from "node-html-parser";
 import { formatDateTime } from "./formatDateService";
+import React from "react";
 
 interface Metadata {
   [key: string]: string;
 }
 
 export interface HtmlTableJsonRow {
   [key: string]: {
-    value: any;
+    value: string | React.JSX.Element;
```

Comment:
maybe use `React.ReactNode` here?
```suggestion
    value: React.ReactNode;
```

Reply:
or the `RenderableNode` type in data-utils since this probably is a tighter type than `ReactNode`

Reply:
I think the issue with this was that it needed to be compatible with `RenderableNode`. Although, now that I'm looking at it again, we could do this instead?

```suggestion
    value: RenderableNode;
```